ch_tools/chadmin/cli/chadmin_group.py

```diff
@@ -50,8 +50,10 @@ def wrapper(ctx, *a, **kw):
             try:
                 cmd_callback(*a, **kw)
                 logging.debug("Command '{}' completed", cmd.name)
-            except Exception:
+            except Exception as e:
+                logging.disable_stdout_logger()
                 logging.exception("Command '{}' failed with error:", cmd.name)
+                logging.print_last_exception(e)
```

Comment:
For another `logging.exception(` calls the behaviour will be as before? Should we log only the latest exc  inside `logging.exception`?

Reply:
`logging.exception` behaviour is not changed except for `backtrace` option which cuts unnecessary part of last stack trace:
```
File "/usr/bin/chadmin", line 8, in <module>
    sys.exit(main())
  File "/opt/yandex/clickhouse-tools/lib/python3.6/site-packages/ch_tools/chadmin/chadmin_cli.py", line 152, in main
    cli.main()
  File "/opt/yandex/clickhouse-tools/lib/python3.6/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/opt/yandex/clickhouse-tools/lib/python3.6/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/yandex/clickhouse-tools/lib/python3.6/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/yandex/clickhouse-tools/lib/python3.6/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/yandex/clickhouse-tools/lib/python3.6/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/opt/yandex/clickhouse-tools/lib/python3.6/site-packages/click/decorators.py", line 26, in new_func
```
`logging.exception` still logs all exceptions in case they are needed

Reply:
Okey, so we don't want to enable it in all cases? 
Also, maybe move it to separate method in logging or put it in the `logging.exception` with option(eg `logging.exception(...., show_all_exceptions)`) ? Think it will much more beautiful:)

Reply:
There is no way to tell if we will need all exceptions or just one, that's why I think we should always log all of them. `logging.exception` is called only once per command, logs will not be flooded with exceptions anyway.

Or did you mean to put `logging.print_last_exception(e)` inside `logging.exception`?

Reply:
>Or did you mean to put logging.print_last_exception(e) inside logging.exception?
Yes